packages/core/test/unit/html/NodeProcessor.data.js

```diff
@@ -291,7 +291,7 @@ module.exports.PROCESS_TAB_HEADER = `
 `;
 
 module.exports.PROCESS_TAB_HEADER_EXPECTED = `
-<tab><template #_header><strong>Lorem ipsum dolor sit amet</strong></template>
+<tab><template #header><strong>Lorem ipsum dolor sit amet</strong></template>
   Header attribute should be inserted as internal _header slot and deleted.
```

Comment:
Maybe the text here can be changed as well for consistency?

Reply:
Thanks for the spot! Have updated the description of the tests.